docs/xlog.md

```diff
@@ -22,21 +22,63 @@
 * Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 -->
 
-# -Xlog 
+# -Xlog
 
-To control logging to the operating system message log, use [`-Xsyslog`](xsyslog.md).
-- When [`-XX:-LegacyXlogOption`](xxlegacyxlogoption.md) is set, which is the default, the `-Xlog` option is processed for compatibility as described in the explanation.
-- When `-XX:+LegacyXlogOption` is set, `-Xlog` is equivalent to `-Xsyslog`.
+This option controls VM logging and is supported for compatibility with the reference implementation. However, only forms of `-Xlog` that enable garbage collection (GC) logging are recognized.
+
+From Eclipse OpenJ9 version 0.24.0, the [`-Xsyslog`](xsyslog.md) option replaces the existing OpenJ9 `-Xlog`  option for message logging to avoid conflicts with the reference implementation. For backward compatibility, you can control the behavior of the `-Xlog` option with the [`-XX:[+|-]LegacyXlogOption`](xxlegacyxlogoption.md) option.
 
 ## Syntax
 
-        -Xlog:gc[:stderr|:file=<filename>]
+        `-Xlog[:<parameters>]`
+
+Without any parameters, this command runs the `-Xsyslog` option and [`-Xverbosegclog`](xverbosegclog.md). Because the reference implementation of `-Xlog` is intended to enable other forms of logging that are not supported by the OpenJ9 VM, the following warning is printed in addition to enabling verbose GC logging:
```

Comment:
-Xlog doesn't "runs the `-Xsyslog` option" when `-XX:-LegacyXlogOption` is set (by default).

Reply:
>-Xlog doesn't "runs the -Xsyslog option" when -XX:-LegacyXlogOption is set (by default).

From the release topic:

"Setting -XX:-LegacyXlogOption (default) causes the VM to process the -Xsyslog option instead and process the -Xlog option only for garbage collection (GC) log requests (-Xlog:gc[:stderr|:file=<filename>])."

Are you objecting to "runs" instead of "processes" or is this deeper than that please?

Reply:
Judging by your next comment, I think we need to update the release topic! What is in there currently is ambiguous. I took this to mean that it will attempt to generate log messages for you (by running -Xsyslog) but also ensure verbose GC logs are included if you are interested in that component. 

I'll have to cascade this change through other topics.

Reply:
-Xlog is either equivalent to -Xsyslog, or it can enable gc logging, it never does both. The change is incompatible, so anybody using the legacy -Xlog will be broken by default on OpenJ9 builds.